Rewrote and added a couple of things.

Greek opened the pull request:
In install.md, the user can refer to the README file if they wanna
install WWT as a browser extension. In README.md, the build command
has been shortened and extra instructions have been added for gadgets

Greek committed:
Rewrote and added a couple of things.

In install.md, the user can refer to the README file if they wanna
install WWT as a browser extension. In README.md, the build command
has been shortened and extra instructions have been added for gadgets

Signed-off-by: Andreas P <me@apap04.com>

MusikAnimal commented on README.md:
```diff
@@ -7,23 +7,29 @@ A browser extension (Chrome and Firefox) based on the https://github.com/wikiwho
 1. Clone the repo
 2. Run `npm install`
-3. Run `./node_modules/.bin/grunt build`
+3. Run `npm build`
```
Should be `npm run-script build` or `npm run build`. This may be dependent on the version of npm, not sure.

Greek replied:
Done.

MusikAnimal commented on tutorials/install.md:
```diff
@@ -17,4 +17,4 @@ Installing WhoWroteThat
 ## As a browser extension
 
-1. Install it. (Details t.b.c.)
+Refer to [the README file](../README.md).
```
Eventually this would be a link to entry on the browser's extension marketplace (Google Web Store, etc.), not how to build it from source.

Greek replied:
Reverted this for now, since this probably won't be released soon on the web store.

MusikAnimal reviewed (changes requested):
Hey, thanks for this! I think there may be some confusion here; it's my understanding that install.md is user-facing documentation on how to install WWT as a gadget or browser extension, that shouldn't involve the build process at all. README.md however has instructions for contributors to the codebase.

Things have changed since install.md was first written; you'll notice the JS and CSS links are broken because we're no longer committing the compiled assets. I think we're still figuring out how we want to expose these files without requiring interested parties build them from source.

As an aside, I think this could have a more description commit summary. "_Rewrote and added a couple of things_" is ambiguous. I thought it involved code changes. Maybe "_Improve installation instructions in README.md_"?

Greek committed:
Improved testing instructions in README.md

Mainly shortened the build command to avoid typing in long paths.
Also reverted a change in install.md. See PR #42.

Signed-off-by: Andreas P <me@apap04.com>

MusikAnimal reviewed (approved):
I think typically if you wanted to test the on-wiki version, you'd just add the script in your userspace rather than create a gadget. But this change to the instructions seems fine for now, at least until we have a proper public-facing build system to copy/paste the compiled assets.
